articles/tokens/refresh-token/current/index.md

```diff
@@ -184,6 +184,8 @@ You should only ask for a new token if the Access Token has expired or you want
 
 Since Refresh Tokens never expire, it is essential to be able to revoke them in case they get compromised.
 
+For the Device Authorization Flow, the only wy to revoke a device authorization is to revoke the Refresh Token assigned to the device. To learn how, see [Remove Devices from Users](/dashboard/guides/users/remove-devices-users).
```

Comment:
```suggestion
For the Device Authorization Flow, the only way to revoke a device authorization is to revoke the Refresh Token assigned to the device. To learn how, see [Remove Devices from Users](/dashboard/guides/users/remove-devices-users).
```
You can revoke device authorization on "Authorized Applications" (it will remove the refresh tokens as well).